<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Davide Bettio &lt;davide.bettio@kdemail.net&gt;</author>
 <class>DateAndTime</class>
 <widget class="QWidget" name="DateAndTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>463</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Date and Time</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <widget class="QCheckBox" name="setDateTimeAuto">
           <property name="text">
            <string>Set date and time &amp;automatically</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="timeServerLabel">
           <property name="text">
            <string>&amp;Time server:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>timeServerList</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="timeServerList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KSeparator" name="kseparator"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KDatePicker" name="cal">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="whatsThis">
            <string>Here you can change the system date's day of the month, month and year.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="timeBox" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <widget class="QQuickWidget" name="timezoneViewer">
      <attribute name="title">
       <string>Time Zone</string>
      </attribute>
     </widget>

    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDatePicker</class>
   <extends>QFrame</extends>
   <header>kdatepicker.h</header>
  </customwidget>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>QLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>setDateTimeAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>cal</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setDateTimeAuto</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
